crates/common/trie/trie.rs

```diff
@@ -103,15 +103,15 @@ impl Trie {
 
```

Comment:
let's keep these changes in another PR. In fact, if we merge the one with the benches first, we can see if this change improves performance.